flink-connector-elasticsearch8/src/main/java/org/apache/flink/connector/elasticsearch/sink/INetworkConfigFactory.java

```diff
@@ -0,0 +1,29 @@
+/*
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+package org.apache.flink.connector.elasticsearch.sink;
+
+import co.elastic.clients.elasticsearch.ElasticsearchAsyncClient;
+
+import java.io.Serializable;
+
+public interface INetworkConfigFactory extends Serializable {
```

Comment:
Why do you need `INetworkConfigFactory`? It is not used anywhere (as far as I can tell), the concrete class `NetworkConfigFactory` is used instead

Reply:
You're right. Just removed it! Thanks